GTG/gtk/backends_dialog.ui

```diff
@@ -149,14 +139,13 @@
               <object class="GtkButtonBox" id="hbuttonbox2">
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
-                <property name="layout_style">edge</property>
                 <child>
                   <object class="GtkButton" id="sync_help">
                     <property name="label">gtk-help</property>
+                    <property name="use_action_appearance">False</property>
```

Comment:
Look at this: https://developer.gnome.org/gtk3/stable/GtkActivatable.html 
Apparently this property has been deprecated (and neither the python function for this is suggested for use in newly written code).